### ISSUE TITLE
Acrylic back panel assembly interference

### ISSUE THREAD

**Issue by BryanHaven** [machineagency/jubilee#65]
During assembly it was noted that core x-y stepper motors interfere with the upper outer screws. 

It was also noted that the tool locking mechanism must be removed to allow panel to be mounted.

**Comment by BryanHaven** [comment]
(https://cdn.discordapp.com/attachments/678103924598898696/688947573020557513/IMG_20200315_231109.jpg)

**Comment by Poofjunior** [comment]
Good catch. I moved those, but it looks like I didn't move them far enough.

**Comment by Poofjunior** [comment]
The screw location is now fixed.
